src/FlaUI.WebDriver.UITests/ActionsTest.cs

```diff
@@ -9,32 +9,44 @@ namespace FlaUI.WebDriver.UITests
     [TestFixture]
     public class ActionsTests
     {
+        private RemoteWebDriver driver;
```

Comment:
I'd like to comply with https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md, so this should be:
```suggestion
        private RemoteWebDriver _driver;
```